docs/Reducers.ts

```diff
@@ -1,17 +1,29 @@
-import { Reducer, combineReducers } from 'redux';
 import { membersReducers, IMembersCompositeState } from './members-example/reducers/MembersReducers';
 import { IReactVaporState } from '../src/utils/ReduxUtils';
 import { lastUpdatedCompositeReducer } from '../src/components/lastUpdated/LastUpdatedReducers';
 import { filterBoxesReducer } from '../src/components/filterBox/FilterBoxReducers';
 import { facetsReducer } from '../src/components/facets/FacetReducers';
+import { perPageCompositeReducer } from '../src/components/navigation/perPage/NavigationPerPageReducers';
+import { paginationCompositeReducer } from '../src/components/navigation/pagination/NavigationPaginationReducers';
+import { loadingsReducer } from '../src/components/loading/LoadingReducers';
+import * as Redux from 'redux';
 
 export interface IReactVaporExampleState extends IReactVaporState {
   membersCompositeState: IMembersCompositeState;
+  lastAction: Redux.Action;
 }
 
-export const Reducers: Reducer<IReactVaporExampleState> = combineReducers<IReactVaporExampleState>({
+const lastAction = (state: IReactVaporExampleState = null, action: Redux.Action): Redux.Action => {
```

Comment:
In the navigation connected example, when we change page or change the per page number, it triggers the loading. I added a simple reducer to check on the last action and dispatch a removeLoading action after a few seconds when the last action is addLoading. It is only used in the context of the example and is not being used in the components themselves.